source/MapPanel.cpp

```diff
@@ -1089,10 +1104,13 @@ void MapPanel::UpdateCache()
 			}
 		}
 
+		vector<string> unmappedSystem = {"map/unexplored-star"};
```

Comment:
```suggestion
		static const vector<string> unmappedSystem = {"map/unexplored-star"};
```
Or make it a constant in the unnamed namespace